package.json

```diff
@@ -26,11 +26,13 @@
     "scoped-http-client": "0.11.0"
   },
   "devDependencies": {
+    "generator-hubot": "^0.3.1",
```

Comment:
This will bring in way too may dependencies. Let's remove this and whatever functionality depends on this.